packages/hoppscotch-app/components/app/PaneLayout.vue

```diff
@@ -6,21 +6,28 @@
       '!flex-row-reverse': SIDEBAR_ON_LEFT && mdAndLarger,
     }"
     :horizontal="!mdAndLarger"
+    @resize="setVerticalPanelInfo"
   >
     <Pane
-      size="75"
+      :size="PANE_MAIN_SIZE.value"
       min-size="65"
       class="hide-scrollbar !overflow-auto flex flex-col"
     >
-      <Splitpanes class="smart-splitter" :horizontal="COLUMN_LAYOUT">
+      <Splitpanes
+        class="smart-splitter"
+        :horizontal="COLUMN_LAYOUT"
+        @resize="setHorizontalPanelInfo"
+      >
         <Pane
-          :size="COLUMN_LAYOUT ? 45 : 50"
+          :size="PANE_MAIN_TOP_SIZE.value"
+          min-size="50"
           class="hide-scrollbar !overflow-auto flex flex-col"
         >
           <slot name="primary" />
         </Pane>
         <Pane
-          :size="COLUMN_LAYOUT ? 65 : 50"
+          :size="PANE_MAIN_BOTTOM_SIZE.value"
```

Comment:
Remove `.value`. Also, we need to consider the `COLUMN_LAYOUT` condition here. Either in the script or in template. I think it should be in script.